assets/pages/digital-subscription-landing/digitalSubscriptionLanding.scss

```diff
@@ -154,13 +154,13 @@
     font-family: $gu-headline;
     list-style-image: svg-url('<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 10 10"><circle cx="5" cy="5" r="5" fill="#ff4e36"/></svg>');
     margin-left: 26px;
+    display: flex;
+    flex-flow: column wrap;
 
     @include mq($from: leftCol) {
-      columns: 2;
-      column-gap: 54px;
-      column-fill: auto;
-      height: 175px;
+      height: 200px;
```

Comment:
As discussed offline, you could use `flex-direction: row` if you wanted to avoid setting `height`.